nixos/modules/installer/cd-dvd/installation-cd-graphical-kde.nix

```diff
@@ -63,11 +34,23 @@ with lib;
       Icon=text-html
     '';
 
+    # Replace default gparted desktop file with one that does "sudo gparted"
```

Comment:
Wouldn't this block be useful for both Gnome and KDE images?

Reply:
Gnome doesn't have files on the desktop anymore.

Reply:
A desktop file is not a file on the desktop, a desktop file is still used to launch applications from launchers etc.

Reply:
But these entries would not be visible any more since gnome doesn't read `~/Desktop`.

Reply:
You suggest this would be useful if it used a patched version of gparted that uses sudo?
Currently here it only copies this file to the desktop.

Reply:
@worldofpeace Yeah, sure. That would be more useful. Especially on an installation media.

Not sure if that's in scope of this PR. But maybe it should be since the behaviour is changed from "I can click that button" to "Clicking that button tells me that it requires root to run".